File: Core java/Day04Assignment/src/com/sonata/employee/Employee.java
package com.sonata.employee;

public class Employee {
public int empid;
public String empname;
public double empsal;

public Employee() {};

public Employee(int id,String name,double sal) {
	this.empid=id;
	this.empname=name;
	this.empsal=sal;

}

void display() {
	System.out.println(this.empid);
	System.out.println(this.empname);
	System.out.println(this.empsal);
	
}


}

File: Typescript assignment and practiced problems/Employee.js
"use strict";
exports.__esModule = true;
exports.Employee = void 0;
var Employee = /** @class */ (function () {
    function Employee(i, n, s) {
        this.empId = 123;
        this.empName = "bhavana";
        this.empsla = 30000;
        this.empId = i;
        this.empName = n;
        this.empsla = s;
    }
    Employee.prototype.display = function () {
        console.log(this.empId);
        console.log(this.empName);
        console.log(this.empsla);
    };
    return Employee;
}());
exports.Employee = Employee;
var e1 = new Employee(124, "bhavana", 34000);
e1.display();
